tests/test_routes.py

```diff
@@ -253,3 +253,210 @@ def test_query_order_list_by_category(self):
     #     order_find_mock.return_value = [MagicMock(serialize=lambda: {'name': 'fido'})]
     #     resp = self.app.get(BASE_URL, query_string='name=fido')
     #     self.assertEqual(resp.status_code, status.HTTP_200_OK)
+
+
+# Copied and pasted TestOrderServer class above
+class TestOrderItemServer(unittest.TestCase):
+    """Order Item Server Tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Run once before all tests"""
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = False
+        # Set up the test database
+        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
+        app.logger.setLevel(logging.CRITICAL)
+        init_db(app)
+
+    @classmethod
+    def tearDownClass(cls):
+        """Run once after all tests"""
+        db.session.close()
+
+    def setUp(self):
+        """Runs before each test"""
+        db.drop_all()  # clean up the last tests
+        db.create_all()  # create new tables
+        self.app = app.test_client()
+
+    def tearDown(self):
+        db.session.remove()
+        db.drop_all()
+
+    def _create_orders(self, count):
+        """Factory method to create orders in bulk"""
+        orders = []
+        for _ in range(count):
+            test_order = OrderFactory()
+            resp = self.app.post(
+                BASE_URL, json=test_order.serialize(), content_type=CONTENT_TYPE_JSON
+            )
+            self.assertEqual(
+                resp.status_code, status.HTTP_201_CREATED, "Could not create test order"
+            )
+            new_order = resp.get_json()
+            test_order.id = new_order["id"]
+            orders.append(test_order)
+        return orders
+
+    def test_index(self):
+        """Test the Home Page"""
+        resp = self.app.get("/")
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(data["name"], "Order Demo REST API Service")
+
+    def test_get_order_list(self):
+        """Get a list of Orders"""
+        self._create_orders(5)
```

Comment:
Get list of all orders, or get list of all order items?

Reply:
I deleted this part and write new code based on the sample code.